src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -903,14 +932,17 @@ ov::pass::NopStridedSlice::NopStridedSlice() {
 
         auto end_node = strided_slice_node->get_input_node_shared_ptr(2);
         if (const auto& end_constant_node = ov::util::get_constant_from_source(end_node)) {
+            int64_t max_value = end_node->get_element_type() == ov::element::i32 ? std::numeric_limits<int32_t>::max()
```

Comment:
Could it be another interger type? i64, u32....?

Reply:
Could we have here element_type::u64 and std::numeric_limits<unt64_t>::max()
std::numeric_limits<int64_t>::max() != std::numeric_limits<uint64_t>::max()

Reply:
yes, you are right
I will improve it in the next PR